docs/index.md

```diff
@@ -1,10 +1,56 @@
 ---
 title: nf-cmgg/structural
-description: A bioinformatics best-practice analysis pipeline for calling structural variants from short reads.
+description: A bioinformatics best-practice analysis pipeline for calling SVs (structural variants), CNVs (copy number variants) and repeat region expansions from short DNA reads
 ---
 
---8<-- "README.md:10:16"
+## Introduction
+
+**nf-cmgg/structural** is a bioinformatics best-practice analysis pipeline for calling SVs (structural variants), CNVs (copy number variants) and repeat region expansions from short DNA reads. The pipeline handles the calling of the variants and postprocessing (filtering, annotating...)
```

Comment:
just use abbreviations, you have declared them higher up

Reply:
This is the start of the documentation site, so I think it should be fully shown here too